index.html

```diff
@@ -9,44 +9,49 @@
 
 </head>
 <body>
-    <section class="section-one">
         <header>
     
-            <div class="header-containt">
-    
-                <nav>
-                    <ul>
-                        <li><a href="#">Portfolio</a></li>
-                        <li><a href="#">About</a></li>
-                        <li><a href="#">Contact</a></li>
-                    </ul>
-    
-                </nav>
-    
-            </div>
+           <!-- Add a navigation icon -->
+          <button class="nav-icon">
+            <i class="fas fa-bars"></i>
+          </button>
+  
+          <!-- Move the navigation list inside a new div -->
+          <div class="nav-list">
+            <nav>
+            <ul>
+              <li><a href="#">Portfolio</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+            </ul>
+            </nav>
+          </div>
         </header>
         <section class="hero-comtain">
           <div class="background-container">     
             <div class="contain">
                 <div>
                    <h1>Hey there. I’m Roland.<br>
                         <span>I’m a software developer</span></h1>
-                        <p> I can help you build a product , feature or website Look through some of my work and<br> experience! If you like what you see and have a project you need coded,<br> 
+                        <p class="hero-paragraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
                             don’t hesitate to contact me.</p>
                 </div>      
             </div>
             <div class="icons">
-                <div><i class="fa-brands fa-github"></i></div>
-                <div><i class="fa-brands fa-linkedin"></i></div>
-                <div><i class="fa-solid fa-hand-peace"></i></div>
-                <div><i class="fa-brands fa-twitter"></i></div>
-                <div><i class="fa-sharp fa-solid fa-m"></i></div>
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+                <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="#"><i class="fa-solid fa-hand-peace"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#"><i class="fa-sharp fa-solid fa-m"></i></a>
```

Comment:
Great done by making this icons links 👍🏼  just add valid links to your social accounts to href properties, and as reequested before please use list items to wrap these items it is good for semantic reasons as ul li tags are for lists